src/main/java/frc/robot/Constants.java

```diff
@@ -18,6 +18,11 @@ public static class IntakeConstants {
     public static final double INTAKE_PERCENTAGE = -0.75;
   }
 
+  public static class ShooterConstants {
+    public static final int SHOOTER_MOTOR_ID = 55;
+    public static final double SHOOTER_MINIMUM_SPEED = 0.1;
```

Comment:
I think we will likely want this as an RPM. 

I am not sure.  If we decide to keep this as a motor output, we should change the variable name

Reply:
Resolved (placeholder value)